src/led.py

```diff
@@ -11,131 +11,54 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Signal states on a LED"""
 
-'''Signal states on a LED'''
-
-import itertools
 import logging
 import os
-import threading
 import time
 
+import aiy.common
 import RPi.GPIO as GPIO
 
-logger = logging.getLogger('led')
+logger = logging.getLogger("led")
```

Comment:
I though we use single quotes for string literals in most of the places. Please keep those as they were.